src/nginx.go

```diff
@@ -2,33 +2,42 @@ package main
 
 import (
 	"fmt"
+	"net/url"
+	"os"
+
 	sdk_args "github.com/newrelic/infra-integrations-sdk/args"
 	"github.com/newrelic/infra-integrations-sdk/data/metric"
 	"github.com/newrelic/infra-integrations-sdk/integration"
 	"github.com/newrelic/infra-integrations-sdk/log"
 	"github.com/newrelic/infra-integrations-sdk/persist"
 	"github.com/pkg/errors"
-	"net/url"
-	"os"
 )
 
 type argumentList struct {
 	sdk_args.DefaultArgumentList
-	StatusURL        string `default:"http://127.0.0.1/status" help:"NGINX status URL."`
-	ConfigPath       string `default:"/etc/nginx/nginx.conf" help:"NGINX configuration file."`
-	RemoteMonitoring bool   `default:"false" help:"Identifies the monitored entity as 'remote'. In doubt: set to true."`
+	StatusURL         string `default:"http://127.0.0.1/status" help:"NGINX status URL. If you are using ngx_http_api_module be sure to include the full path ending with the API version number"`
+	ConfigPath        string `default:"/etc/nginx/nginx.conf" help:"NGINX configuration file."`
+	RemoteMonitoring  bool   `default:"false" help:"Identifies the monitored entity as 'remote'. In doubt: set to true."`
+	ConnectionTimeout int    `default:"1" help:"OHI connection to Nginx timeout in seconds"`
+	StatusModule      string `default:"discover" help:"Name of Nginx status module. discover | ngx_http_stub_status_module | ngx_http_status_module | ngx_http_api_module"`
```

Comment:
is 'discover' the correct default value?

Reply:
Yes, it keeps the previous behaviour to avoid breaking changes